Adjustments and reworking some layouts (#213)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_deploying-smart-gateways.adoc

. Edit the `default` ServiceTelemetry object and add a `clouds` parameter with your configuration:
+
[WARNING]
====

> Long cloud names can cause the maximum pod name of 63 characters to be exceeded. The combination of the `ServiceTelemetry` name (i.e. `default`) and the `clouds.name` should not exceed 19 characters. Topic addresses have no such limitation and can be different than the `clouds.name` value.

Reviewer: ```suggestion
Long cloud names can cause the maximum pod name of 63 characters to be exceeded. Ensure that the combination of the `ServiceTelemetry` name `default` and the `clouds.name` does not exceed 19 characters. Topic addresses have no character limitation and can be different than the `clouds.name` value.
```
